pkg/bbgo/indicator_set.go

```diff
@@ -3,7 +3,10 @@ package bbgo
 import (
 	"github.com/sirupsen/logrus"
 
-	"github.com/c9s/bbgo/pkg/indicator/v2"
+	indicatorv2 "github.com/c9s/bbgo/pkg/indicator/v2"
+	"github.com/c9s/bbgo/pkg/indicator/v2/momentum"
+	"github.com/c9s/bbgo/pkg/indicator/v2/trend"
+	"github.com/c9s/bbgo/pkg/indicator/v2/volatility"
```

Comment:
some external strategy repositories are importing the indicators in the original way, we're trying not to break them.

if you don't mind, could you please put the indicators in the v2 for now? and maybe later we will come up a new idea & new design for categorizing these files~

it would be great if you can join the discussion with us :D